Assets/Scripts/Interactables/CocktailMiniGame.cs

```diff
@@ -259,6 +276,11 @@ private void GiveCorrespondingDrink()
                     //for when we will have the specific sprite icon
                     // itemIcon = GetCocktailIcon(cocktailItemType.Value)
 
+                    //OR?
```

Comment:
Comment?

Reply:
i left legacy code to see which option worked best, will remove uncecessary stuff x)

Reply:
removed